container/secrets.go

```diff
@@ -81,5 +82,16 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	baseDir := "/usr/share/rhel/secrets"
+	overrideDir := "/etc/containers/secrets"
```

Comment:
`baseDir` was already defined at the top of this file and `overrideDir` could have been defined there as well saving those local vars